src/common/pages/profile/userOpTable.tsx

```diff
@@ -102,6 +107,7 @@ const TransactionOperationTable = (props: any) => {
                       <tbody>
                         {Object.keys(opVal).map((key, k: number) => {
                           k = k + Math.floor(Math.random() * 10000) + 9000;
```

Comment:
don't use randomness for keys, use unique values from array/object in equation.

Reply:
All the random keys with random numbers have been resolved